src/Haskell/Ide/Engine/Server.hs

```diff
@@ -855,22 +854,23 @@ requestDiagnostics DiagnosticsRequest{trigger, file, trackingNumber, documentVer
       forM_ dss $ \(pid,ds) -> do
         debugm $ "requestDiagnostics: calling diagFunc for plugin:" ++ show pid
         let
+          pid' = coerce pid
```

Comment:
Leaves me uncomfortable, as before.
If we decide to keep `coerce`, perhaps add a signature to this function, to document what the conversion is.